### DISCUSSION TITLE
Are handling a touch command and taking a screenshot to the SD card mutually exclusive?

### DISCUSSION THREAD

**Discussion by Carlo47** [witnessmenow/ESP32-Cheap-Yellow-Display#143] [Q&A]
I can read and display bitmaps or jpg images from the SD card but I am not able to save a screenshot to the SD card by a touch command, i.e. either I can save the screenshot and the touch screen does not respond or I can process the touch input but cannot save a screenshot. I use the lovyanGFX library and my example project can be found here: https://github.com/Carlo47/CYD_Basics. Can anyone help?

**Comment by clach04** [comment]
Cross reference SoftSPI https://github.com/witnessmenow/ESP32-Cheap-Yellow-Display/discussions/88
